modules/network-observability-config-flows-at-ingestion.adoc

```diff
@@ -0,0 +1,76 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/configuring-operators.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-config-flows-at-ingestion_{context}"]
+
+= Filtering network flows at ingestion
+
+You can create filters to reduce the number of generated network flows. Filtering network flows can help reduce the resource usage of the Network Observability Operator.
+
+There are two different kinds of filters that you can configure:
+
+[id="ebpf-agent-filters_{context}"]
+== eBPF agent filters
+
+eBPF agent filters maximize performance because they take effect at the earliest stage of the Network Observability Operator process.
```

Comment:
same here, it's not about the operator.
I'd suggest this:
```suggestion
eBPF agent filters maximize performance because they take effect at the earliest stage of the network flows collection process.
```